source/main.cpp

```diff
@@ -314,7 +319,8 @@ void GameLoop(PlayerInfo &player, const Conversation &conversation, const string
 		
 		GameWindow::Step();
 		
-		timer.Wait();
+		if(testContext.testToRun.empty())
```

Comment:
Please include a comment, e.g.
```c++
// Run automated tests as quickly as possible.
```

Reply:
Good idea. Added a comment about this (and clarified a related comment).